src/redis_client.py

```diff
@@ -110,15 +115,92 @@ def get_top_n_filtered_results(redis_db,
     return pipeline_score_list
 
 '''
```

Comment:
None of these functions should be inside the redis_client
Redis_client is an abstraction over the redis client that makes it easier for Throttlebot to interface with Redis. The plotting should be in another function, maybe a visualizer.py file?